docs/install.qmd

```diff
@@ -62,10 +62,58 @@ Deltares also provides the iMOD QGIS plugin, which can be used to inspect the re
 For larger edits using Python is recommended.
 One can also use Ribasim Python to build entire models from base data, such that your model setup is fully reproducible.
 
-Users can choose to use Ribasim Python or QGIS plugin or a combination of them to build a Ribasim model.
-And then use the Ribasim core to run the simulation.
+# Install all components into a Pixi workspace (recommended) {#sec-install-all}
 
-# Install Ribasim core
+We recommend most users to install all components into one [Pixi](https://pixi.sh/) workspace.
+With it we can provide quick and reproducible steps to get started with modeling.
+Note that if you only wish to visualize models, you can [just install the QGIS plugin](@sec-install-qgis-plugin).
+
+First install Pixi following the [Pixi installation documentation](https://pixi.sh/latest/installation/).
+Note that if the recommended installation methods don't work due to restriction on your PC, the [zipped executable](https://pixi.sh/latest/installation/#download-from-github-releases) is likely to still work.
+
+```sh
+# Open your terminal in a new empty project folder.
+# Create a new Pixi workspace.
+pixi init
+
+# Add Ribasim, QGIS and download & unpack tools.
+# Remove qgis if you prefer to use an existing QGIS installation.
+pixi add ribasim==2025.4.0 gh 7zip qgis
```

Comment:
Perhaps when the `pixi task` CLI gets more complete, we can define more of these commands as tasks that depend on each other, and take the Ribasim version as arguments. Then users can use those tasks later on to update Ribasim as well.

We could possibly also use some of the QGIS profile tricks we do in our dev setup to install plugins automatically, and make `pixi run qgis` use an isolated profile. But that quickly becomes a lot more complicated because you have to run Python code, and get that on the system and such, and users might prefer to use the default profile anyway.